tests/Domain/Fixture/FixtureCustomizerTest.php

```diff
@@ -0,0 +1,147 @@
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+class FixtureCustomizerTest extends TestCase {
+
+  /**
+   * @var \Acquia\Orca\Helper\Filesystem\FinderFactory|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private FinderFactory|ObjectProphecy $finderFactory;
+
+  /**
+   * @var \Symfony\Component\Filesystem\Filesystem|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private Filesystem|ObjectProphecy $filesystem;
+
+  /**
+   * @var \Acquia\Orca\Helper\Filesystem\FixturePathHandler|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private FixturePathHandler|ObjectProphecy $fixturePathHandler;
+
+  /**
+   * @var \Symfony\Component\Console\Output\OutputInterface|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private OutputInterface|ObjectProphecy $output;
+
+  /**
+   * @var \Acquia\Orca\Options\FixtureOptions|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private ObjectProphecy|FixtureOptions $fixtureOptions;
+
+  private const SUT_IS_PERZ = 'drupal/acquia_perz';
+  private const SUT_IS_NOT_PERZ = 'drupal/example';
+
+  /**
+   * @var \Acquia\Orca\Domain\Package\Package|\Prophecy\Prophecy\ObjectProphecy
+   */
+  private ObjectProphecy|Package $package;
+
+  public function setUp(): void {
+    $finder = $this->prophesize(Finder::class);
+    $finder
+      ->in(Argument::any())
+      ->willReturn($finder);
+    $finder
+      ->contains(Argument::any())
+      ->willReturn($finder);
+
+    $file = $this->prophesize(SplFileInfo::class);
+    $file->__toString()
+      ->willReturn(Argument::type('string'));
+    $fileIterator = new \ArrayIterator([$file->reveal()]);
+
+    $finder
+      ->getIterator()
+      ->willReturn($fileIterator);
+
+    $this->finderFactory = $this->prophesize(FinderFactory::class);
+    $this->finderFactory
+      ->create()
+      ->willReturn($finder);
+
+    $this->fixturePathHandler = $this->prophesize(FixturePathHandler::class);
+    $this->fixturePathHandler
+      ->getPath(Argument::type('string'))
+      ->willReturn(Argument::type('string'));
+
+    $this->output = $this->prophesize(OutputInterface::class);
+
+  }
+
+  public function createCustomizer(): FixtureCustomizer {
+    $finderFactory = $this->finderFactory->reveal();
+    $fileSystem = $this->filesystem->reveal();
+    $fixturePathHandler = $this->fixturePathHandler->reveal();
+    $output = $this->output->reveal();
+    return new FixtureCustomizer($finderFactory, $fileSystem, $fixturePathHandler, $output);
+  }
+
+  public function testParagraphsRemovalWhenSutIsNotPerz(): void {
+
+    $this->package = $this->prophesize(Package::class);
+    $this->package
+      ->getPackageName()
+      ->willReturn(self::SUT_IS_NOT_PERZ);
+    $this->fixtureOptions = $this->prophesize(FixtureOptions::class);
+    $this->fixtureOptions
+      ->getSut()
+      ->willReturn($this->package);
+
+    $this->filesystem = $this->prophesize(Filesystem::class);
+    $this->filesystem
+      ->remove(Argument::any())
+      ->shouldBeCalled();
```

Comment:
Are you sure this test is effectively covering the test case? Basically everything is mocked, so we're kind of comparing our assumptions _to_ our assumptions. What would it take for this test to break? If you deliberately broke the functionality, would this test fail? Same questions for the next two test methods.

Reply:
I have tested this and found that tests are failing as expected if we try and break the code. So they are working fine.

But it would be great if you can suggest changes to improve at least one of the tests then I will incorporate those improvements to other tests as well.